internal/eval/comparable.go

```diff
@@ -0,0 +1,9 @@
+package eval
+
+import "github.com/cedar-policy/cedar-go/types"
+
+type ComparableValue interface {
+	types.Value
+	LessThan(types.Value) (bool, error)
+	LessThanEqual(types.Value) (bool, error)
```

Comment:
Can we change LessThanEqual to LessThanOrEqual, that better matches the cedar docs:

https://docs.cedarpolicy.com/policies/syntax-operators.html#function-lessThanOrEqual